public/locales/fr/translation.json

```diff
@@ -305,7 +305,8 @@
       },
       "minifiche": {
         "display-on-click": "Afficher la minifiche",
-        "template": "Modèle de la minifiche"
+        "template": "Modèle de la minifiche",
+        "highlight-color": "Choisir la couleur de surbrillance"
```

Comment:
Can you add the verb that describes the action to the key?
Could you also change the key "minifiche" 3 lines above by a less french one?